js/area/signin/signin.js

```diff
@@ -303,7 +304,7 @@ SignInComponent.prototype.switchPanel = function (panelClass) {
 	newPanel
 		.attr('aria-hidden', 'false')
 		.find('[tabindex]')
-		.removeAttr('disabled')
+		.prop('disabled', false)
```

Comment:
This would give us a mix of `.prop('disabled', false)` and `attribute('disabled', 'disabled')` on `oldPanel` is that intentional? See L316.